README.md

```diff
@@ -485,6 +486,50 @@ To run:
 `scripts/portals-unblock-skylinks.sh -e @my-vars/skylinks-unblock.yml --limit eu-fin-1`  
 `scripts/portals-unblock-skylinks.sh -e @my-vars/skylinks-unblock.yml --limit webportals_prod`
 
+### Block and Unblock Incoming Traffic to Portals
+
+Playbook:
+
+- Blocks incoming traffic from IPs or IP ranges defined in
+  `private_vars.incoming_ips_ip_ranges_block` list.
+- Unblocks previously blocked incoming traffic from IPs or IP ranges defined in
+  `private_vars.incoming_ips_ip_ranges_unblock` list.
+
+Preparation:  
+`private_vars_file` defined in `webportals.yml` defines a filepath where
+private variables are stored. Default path is in `ansible-private` repository,
+in `private-vars/private-vars.yml` file. In this file there are defined 2 lists
+of IPs or Ip ranges: `incoming_ips_ip_ranges_block` and
+`incoming_ips_ip_ranges_unblock`. To block an IP/IP range, add it to the block
+list, to unblock the previously blocked IP/IP range, remove it from the block
+list and add it to the unblock list and execute the playbook.
```

Comment:
So, if I want to block a new range I need to add it to the list. After the range is blocked, we would generally remove it from the list, then?

I wonder if it wouldn't be better to keep all the blocked ranges in that list, so we can easily apply them to newly created servers. This could make the list very very long and hard to work with, though.

Can we relatively easily sync up all machines on the list of IP ranges they block?

Reply:
> So, if I want to block a new range I need to add it to the list. After the range is blocked, we would generally remove it from the list, then?
> 

No, we add the range to the list, we block it and we keep it in the list.

We remove the range from the block list and add it to unblock list only in case we want to unblock it later.

> I wonder if it wouldn't be better to keep all the blocked ranges in that list, so we can easily apply them to newly created servers. This could make the list very very long and hard to work with, though.
> 

Yes, we keep blocked ranges in the list, so they are applied also to newly created servers.

> Can we relatively easily sync up all machines on the list of IP ranges they block?

See above.